chore(release): v3.20.1

sfoster1 opened the pull request:
3.20.0 fixes #6394, which causes robots upgraded from 3.18.0 or previous directly to 3.20.0 to be unable to upload protocols until labware calibrations are cleared.

Laura-Danielle committed:
fix(api, app): Ensure index file exists before reading from it (#6410)

A bug has been popping up for users upgrading from a server version other than 3.19.0. Previously, when the index file does not exist, but a labware calibration file does, the migration check for the index file would fail because there are no protections to ensure that we are reading from a file that exists.

We also need to defer loading calibration data to after protocol uploads complete, since calibration data may not be valid until the simulation is done.

closes #6394

sfoster1 committed:
chore(release): v3.20.1-alpha.0 (#6432)

First alpha for 3.20.1, including the fix for #6394

bndo reviewed (dismissed):
LGTM.

nusrat813 reviewed (approved):
Tested - working as intended

sfoster1 committed:
chore(release): 3.20.1 (#6437)

3.20.1 is a robot stack release that fixes an issue in 3.20.0 where protocols
cannot be uploaded until labware calibrations are reset, if the robot
was updated from a version at or under 3.18.0.

bndo reviewed (approved):
Looks good. 🕵️
